core/unit_test/CMakeLists.txt

```diff
@@ -77,4 +77,6 @@ macro(Cabana_add_tests)
 endmacro()
 
 Cabana_add_tests(NAMES AoSoA Slice DeepCopy Tuple Sort LinkedCellList NeighborList Parallel)
-Cabana_add_tests(MPI NAMES CommunicationPlan Distributor Halo)
+if(Cabana_ENABLE_MPI)
```

Comment:
OK that's the correct fix.

Reply:
Yeah, the first fix wasn't more a workaround.